src/api/spec/controllers/webui/main_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Webui::MainController do
+  let(:user) { create(:confirmed_user) }
```

Comment:
Talked with henne now. We don't want to use the backend in the rspec test suite.

So no need to use static data here as long as we don't have to record any backend calls with VCR.